AlumniConnect/views.py

```diff
@@ -85,22 +88,50 @@ def signup(request):
             roll_no = request.POST['username'] # username and roll_no are same
             
             # user created using form
-            user = form.save()
-            user.is_active = False 
-            user.save()
+            user = User.objects.create_user(
+                username = form.cleaned_data.get('username'),
+                password = form.cleaned_data.get('password'),
+                email = form.cleaned_data.get('email'),
+                is_active = False
+                )
             
             # now making profile for user
             profile = Profile(user=user, roll_no=roll_no, role=role)
             profile.save()
             
-            return HttpResponse("Sign Up successfully done")
+             # sending mail for activation
+            current_site = get_current_site(request)
+            from_email = settings.DEFAULT_FROM_EMAIL
+            to = [user.email]
+            subject = "[noreply] SAC Account Activation"
+            html_message = render_to_string('AlumniConnect/account_activation_email.html', {
+                'user':user,
+                'domain':current_site,
+                'uid':urlsafe_base64_encode(force_bytes(user.pk)),
+                'token':account_activation_token.make_token(user)
+            })
+            plain_message = strip_tags(html_message)
+            send_mail(
+                subject = subject,
+                message = plain_message,
+                from_email = from_email,
+                recipient_list=to,
+                html_message = html_message,
+                fail_silently=False,
+            )
+
+            return render(request,"AlumniConnect/confirm_email.html")
```

Comment:
Always redirect after complete a request, other on refreshing it will show "Submit the form again" pop-up.